src/app/features/tech-record/components/tech-record-title/tech-record-title.component.ts

```diff
@@ -31,6 +31,10 @@ export class TechRecordTitleComponent implements OnInit {
     return this.vehicleTechRecord?.vrms.find(vrm => vrm.isPrimary === true)?.vrm;
   }
 
+  get vehicleType(): string {
+    return this.vehicleTechRecord?.techRecord[0].vehicleType!;
```

Comment:
what if we had changed the vehicle type later? This feels unsafe

Reply:
I meant to change this, ty for catching

Reply:
This should be omitted altogether.